.github/workflows/actions.yml

```
@@ -0,0 +1,44 @@
+name: Continuous Integration
+on:
+  push:
+    path-ignore:
+      - docs/README.md
+  pull_request:
+    branches:
+      - master
+    path-ignore:
+      - docs/README.md
+
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: docker build --target test --tag mytest .
+    - run: docker run --env-file .env.test mytest
+  prod-check:
+    if: ${{ github.ref == 'refs/heads/master' }}
+    runs-on: ubuntu-latest
+    steps:
+    - run: echo "Deploying to production server on branch $GITHUB_REF"
+
+    - name: Login to DockerHub
+      uses: docker/login-action@v1 
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }} #contained in GitHub
+        password: ${{ secrets.DOCKERHUB_TOKEN }} #contained in GitHub
+      name: Build and push
```

If you look, your latest build hasn't run successfully because the yml isn't valid; currently this property is still tied to the "Login to DockerHub" step - you need to start a new step with a `-`:
```suggestion
    - name: Build and push
```